go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/canonical/edgex-snap-hooks
+module github.com/canonical/edgex-snap-hooks/v2
 
```

Comment:
Should we be checking in `go.sum`?

Reply:
yes, we probably should , as per these [best practice](https://github.com/golang/go/wiki/Modules#should-i-commit-my-gosum-file-as-well-as-my-gomod-file) instructions. We should then also remove go.sum from .gitignore.